esx_service/vmdk_ops.py

```diff
@@ -606,6 +606,13 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     elif cmd == "remove":
         response = removeVMDK(vmdk_path)
+        # remove succeed, remove infomation of this volume from volumes table
```

Comment:
> The reason we cannot insert it into "volumes" table...

Ah, yet another reason to have _fallback_ tenant then.

> I don't understand what do you mean "how does it impact the location of DB code ?". 

My point was that the code removing volume from the table is in the wrong place. It should NOT be in executeRequest, it is not the correct layer. I was not sure how your comment was addressing mine :-)